### ISSUE TITLE
Binder error when using nix-env

### ISSUE THREAD

**Issue by FRidh** [FRidh/nix-tutorials#7]
Installing anything with `nix-env` in binder fails
```
error: while setting up the build environment: unable to make '/' private mount: Permission denied
```

@costrouc maybe you have an idea? Profiles are not set up I suppose?

**Comment by FRidh** [comment]
Fails when building. Inserting into the store is fine. `nix-env -i` fails because it also builds a derivation.

The sandbox needs to be disabled
```
nix-build --option sandbox false
```
Installation I think creates a `/etc/nix.conf` file. We should disable sandboxing in there.

**Comment by FRidh** [comment]
fixed with 9de5adce1c6016e6faecdeb326a3b3f081c5e048
